SConstruct

```diff
@@ -72,6 +73,7 @@ env_base.disabled_modules = []
 env_base.use_ptrcall = False
 env_base.module_version_string = ""
 env_base.msvc = False
+env_base.stable_release = version.status == "stable"
```

Comment:
I made this an `env` variable as we might want to do more stable-specific logic in the future, especially in `platform/*/detect.py`.

Future-proofing like this is [against our guidelines](https://docs.godotengine.org/en/latest/community/contributing/best_practices_for_engine_contributors.html#the-problem-has-to-exist), but in this case I think it's worth it ;)